<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>780</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Удалить запись</string>
   </property>
   <widget class="QLineEdit" name="DeleteMaterialID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>номер записи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteMaterialBtn">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>780</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить запись</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="AddMaterialName">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>название материала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AddMaterialSquare">
       <property name="placeholderText">
        <string>площадь поры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AddMaterialSquareDeviation">
       <property name="placeholderText">
        <string>откл. от площ.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AddMaterialPorosity">
       <property name="placeholderText">
        <string>пористость</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AddMaterialPorosityDeviation">
       <property name="placeholderText">
        <string>откл. от порист.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddMaterialBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="MaterialsTableview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>781</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
